core/src/main/java/org/apache/accumulo/core/fate/accumulo/AccumuloStore.java

```diff
@@ -82,7 +85,9 @@ protected Stream<FateIdStatus> getTransactions() {
       scanner.setRange(new Range());
       TxColumnFamily.STATUS_COLUMN.fetch(scanner);
       return scanner.stream().onClose(scanner::close).map(e -> {
-        return new FateIdStatus(parseTid(e.getKey().getRow().toString())) {
+        FateId fateId =
+            FateId.from("FATE:" + fateInstanceType + ":" + e.getKey().getRow().toString());
```

Comment:
Could have a `FateId.from(FateInstanceType it, String hexId)` method and do the following 
```suggestion
            FateId.from(fateInstanceType, e.getKey().getRow().toString());
```

Reply:
Yes, good suggestion